styles/whatsapp-web/catppuccin.user.css

```diff
@@ -60,7 +65,7 @@
     @crust: @catppuccin[@@lookup][@crust];
     @accent-color: @catppuccin[@@lookup][@@accent];
 
-    .dark {
+    & {
```

Comment:
This is redundant and can be removed, also remove the closing tag, thanks.
```suggestion
```

Reply:
How about now? This creates quite a large patch because of the indentation change, is that ok?

Reply:
That's alright. We would rather consistency then not.